src/accelerate/commands/launch.py

```diff
@@ -61,340 +61,430 @@
 
 logger = logging.getLogger(__name__)
 
+_options_to_group = {
+    "--multi-gpu": "Distributed GPUs",
+    "--tpu": "TPU",
+    "--use_mps_device": "MPS",
+    "--use_deepspeed": "DeepSpeed",
+    "--use_fsdp": "FSDP",
+    "--use_megatron_lm": "Megatron-LM",
+}
+
+
+def _clean_option(option):
```

Comment:
I hate this underscore convention in Python as it hurts readability and nothing is ever truly private. In Accelerate, like for Transformers, the convention is that things that are not in the main init are private. So no need to add all those _ :-)

Reply:
Made public! :) Will note down for next time